test-data/config.toml

```diff
@@ -1,6 +1,7 @@
+[settings]
 album_art_pattern = '^Folder\.(png|jpg|jpeg)$'
 
-[[mount_dirs]]
+[[settings.mount_dirs]]
```

Comment:
TODO:
Consider pulling `mount dirs` and `DDNS settings` out of `settings`. Would keep config file format unchanged and the structure would map more accurately to the underlying tables.